server/odc-service/src/main/java/com/oceanbase/odc/service/connection/ConnectionTesting.java

```diff
@@ -217,13 +217,13 @@ private ConnectionConfig reqToConnectionConfig(TestConnectionReq req) {
         } else {
             config.setType(req.getType());
         }
+        config.setDefaultSchema(req.getDefaultSchema());
```

Comment:
after ODC v4.2.0, datasource does not has 'defaultSchema' concept for oceanbase connectType

Reply:
just rolled back the previous modification operation in this way

Reply:
what is the reason to  move from line 226 to line 220 ?